packages/cli-test/src/cli/cli-process.ts

```diff
@@ -49,7 +49,7 @@ export class SlackCLIProcess {
   /**
    * @description The CLI command to invoke
    */
-  public command: string;
```

Comment:
One part of the breaking change: commands are now composed of an array of strings, representing the space-delimited shell invocation.

Reply:
I think this is a learning for all of our back-pockets. When dealing with CLI arguments in code, it's best to represent them as an array rather than a single string. Command-line frameworks like Cobra also always parse the raw command string into an array as well and it's likely because it's a more reliable way to interact with CLI arguments in code.